Makefile.toml

```diff
@@ -182,6 +182,8 @@ private = true
 script = [
 '''
 docker pull parity/parity:v1.11.11
+docker pull parity/parity:v2.1.3
```

Comment:
❌Instead of pulling both, we should rather get rid of one of them... don't you think so?

Reply:
This is supposed to be a quick fix to get the build stable.

The getting rid of is tracked in the testcontainers-rs issue quoted in the description.

Reply:
Would this be unnecessary once we do #606?

Reply:
It would still be necessary to pull bitcoin and parity images for the e2e tests.